### DISCUSSION TITLE
Fallback stream using same path

### DISCUSSION THREAD

**Discussion by smierd** [bluenviron/mediamtx#3471] [Questions]
### Question

**Version**
1.6.0

**Operating System**
Windows 11

**Question**
I am trying to have a single path adapt to various situations. I have one source that publishes to MediaMTX and another source that I must read from. I want to dynamically change which feed is published on the one path depending on the state of either source. If the publishing source is not turned on, I would like it to fall back to the second source via RTSP using the same path.

RTSP source 1: publishing to Media MTX
RTSP source 2: RTSP address 

Is such a thing possible?

**EDIT**
disregard, the fallback argument mostly solves my problems. Would be nice if the when the publisher comes back online, it kicks out the fallback stream but it doesn't do that and only resumes if the fallback goes offline.

**Comment by aler9** [comment]
The ability to show a fallback stream without disconnecting readers is being tracked here: #2214
